examples/qml_video/main.cpp

```diff
@@ -110,6 +110,7 @@ int main(int argc, char *argv[])
             pf = QVideoFrame::Format_NV12;
             break;
         default:
+            pf = QVideoFrame::Format_NV12;
```

Comment:
what does frame.frame()->format show?

Reply:
Format_Invalid

Reply:
AV_PIX_FMT_NONE ?

Reply:
Sorry, logged wrong info.. 

frame.frame()->format:  174

Reply:
might be better to add case AV_PIX_FMT_D3D11: ?

Reply:
sure, sorry for late response :)